NitroxClient/Communication/Packets/Processors/VehicleUndockingProcessor.cs

```diff
@@ -56,7 +56,7 @@ private void StartVehicleUndocking(VehicleUndocking packet, GameObject vehicleGo
                 vehicle.mainAnimator.SetBool("player_in", true);
                 playerInstance.Attach(vehicle.playerPosition.transform);
                 // It can happen that the player turns in circles around himself in the vehicle. This stops it.
-                playerInstance.RigidBody.angularVelocity = Vector3.zero;
+                // playerInstance.RigidBody.angularVelocity = Vector3.zero; // May not happen anymore needs verifying
```

Comment:
What does the verifying say?

Reply:
Gotta get a test session going to repeatedly test this as judging by the original comment it doesnt seem to happen often?